packages/components-react/projects/nextjs/.gitignore

```diff
@@ -24,6 +24,8 @@
 styles/getSharedStyles.tsx
 # sub directories only, because we have layout.tsx, page.tsx and providers.tsx in app folder
 app/*/
+!app/modal-route-change
```

Comment:
Can't we define these in the `pages` directory so we don't mix up generated code with manually written stuff?